docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,68 @@
+# Experimental Features
+
+Cluster API now ships with a new experimental package that lives under exp/ directory which has new features. This is a
+temporary location for features which will be moved to their permanent locations after graduation. Users can experiment with these features by enabling them using feature gates.
+
+## Enabling Experimental Features for Management Clusters Started with clusterctl
+
+Users can enable/disable features by setting OS environment variables before running `clusterctl init`, e.g.:
+
+```yaml
+export EXP_CLUSTER_RESOURCE_SET=true
+
+clusterctl init --infrastructure vsphere
+```
+
+As an alternative to environment variables, it is also possible to set variables in the clusterctl config file located at `$HOME/.cluster-api/clusterctl.yaml`, e.g.:
+```yaml
+# Values for environment variable substitution
+EXP_CLUSTER_RESOURCE_SET: "true"
+```
+In case a variable is defined both in the config file and as an OS environment variable, the latter takes precedence.
+For more information on how to set variables for clusterctl, see [clusterctl Configuration File](../../clusterctl/configuration.md)
```

Comment:
Should we also add a note about enabling the flag on an existing management cluster? ie. modifying the controller flag in the deployment

Reply:
Yep, added.